packages/web/modals/review-order.tsx

```diff
@@ -27,7 +28,7 @@ import {
   useWindowSize,
 } from "~/hooks";
 import { isValidNumericalRawInput } from "~/hooks/input/use-amount-input";
-import { QuoteType, useSwap } from "~/hooks/use-swap";
```

Comment:
Nit: I prefer something more specific like `Quote` over `QuoteType`. We know it's a type.